Documentation/coding-guidelines/breaking-change-rules.md

```diff
@@ -194,6 +194,9 @@ successfully bind to that overload, if simply passing an `int` value. However, i
 
 * Adding or removing `static` keyword from a member
 
+* Adding a private field to a struct that previously had no state (unless the type of the added field is also a struct with no state)
```

Comment:
It doesn't have to be a private field it can be any field. I think I would just remove the part about "unless it the type of added field is also stateless", it highly unlikely and probably should still be avoided.

>"it could cause ‘CS0165: Use of unassigned local variable’ in code that uses it." 

You should call out the reason for the error and not just the error. The reason is that you could end up with code that now has uninitialized data. This is both potentially a source breaking and binary breaking change.